lib/decrypt.rb

```diff
@@ -0,0 +1,13 @@
+require_relative './enigma'
```

Comment:
Required enigma in order to be able to create an instance of enigma to call encrypt/decrypt method